source/jsonnet/england-wales/household/blocks/who-lives-here/anyone_else.jsonnet

```diff
@@ -0,0 +1,177 @@
+local rules = import '../../../lib/common_rules.libsonnet';
+local placeholders = import '../../lib/placeholders.libsonnet';
+
+local questionTitle(hasPrimary) = (
+  if hasPrimary then {
+    text: 'Do any of the following people also live at {household_address} on {census_date}?',
+    placeholders: [
+      placeholders.address,
+      placeholders.censusDate,
+    ],
+  } else {
+    text: 'Do any of the following people live at {household_address} on {census_date}?',
+    placeholders: [
+      placeholders.address,
+      placeholders.censusDate,
+    ],
+  }
+);
+
+local anyoneElseAnswerExclusiveText(hasPrimary) = (
+  if hasPrimary then {
+    label: 'None of these apply, I am the only person who usually lives here',
+    value: 'None of these apply, I am the only person who usually lives here',
+  } else {
+    label: 'None of these apply, no-one usually lives here',
+    value: 'None of these apply, no-one usually lives here',
+    description: 'For example, this is a second address or holiday home',
+  }
+);
+
+local question(hasPrimary) = {
+  type: 'MutuallyExclusive',
+  id: 'anyone-else-driving-question',
+  title: questionTitle(hasPrimary),
+  definitions: [
+    {
+      title: 'Who to include',
+      contents: [
+        { description: 'This is often their permanent or family home.' },
+        {
+          description: {
+            text: 'If someone has more than one home address, include them at this address if they generally spend most of their time here. For example, children whose parents live apart. If they split their time equally then include them at this address if they are staying overnight on {census_date}.',
+            placeholders: [
+              placeholders.censusDate,
+            ],
+          },
+        },
+        {
+          description: 'Remember to include',
+        },
+        {
+          list: [
+            '<strong>students</strong>, if this is either their term-time or their out of term-time address',
+            '<strong>people with a second address for work</strong>, if this is their home address',
+            '<strong>armed forces members</strong>, if this is their home address',
+          ],
+        },
+
+      ],
+    },
+  ],
+  mandatory: true,
+  answers: [
+    {
+      label: 'Select all that apply. You can select more than one for each person',
+      id: 'anyone-else-driving-question-answer',
```

Comment:
Any reason to suffix the answer id with `question` and `answer`?. I think normally just suffix the id with the type it's for. Not set guidelines for these.

Reply:
Only because anyone-else-answer is already in use as an ID

Reply:
But this one would be `anyone-else-driving--answer`?

Reply:
yeah Lewis commented on this - open to other suggestions